openssl/src/ssl/mod.rs

```diff
@@ -613,6 +613,19 @@ impl SslContext {
     }
 
     #[allow(non_snake_case)]
+    /// Advertise the CAs from a file as being accepted for client certificates.
+    ///
+    /// This does not actually make the certificates trusted. You'll still need to call set_CA_file
+    /// for that.
+    pub fn advertise_client_CA_list<P: AsRef<Path>>(&mut self, file: P) -> Result<(), SslError> {
+        let file = CString::new(file.as_ref().as_os_str().to_str().expect("invalid utf8")).unwrap();
+        wrap_ssl_result(unsafe {
+            let names = ffi::SSL_load_client_CA_file(file.as_ptr() as *const _);
```

Comment:
This'll need to be run through 2 rounds of error checking, or we'll pass a null pointer to ssl_ctx_set_client_ca_list